src/libtls/tls_peer.c

```diff
@@ -1199,8 +1199,7 @@ static status_t send_client_hello(private_tls_peer_t *this,
 								  bio_writer_t *writer)
 {
 	tls_cipher_suite_t *suites;
-	bio_writer_t *extensions, *curves = NULL, *versions, *signatures;
-	bio_writer_t *key_share;
+	bio_writer_t *extensions, *curves = NULL, *versions, *signatures, *key_share;
```

Comment:
Hehe, this will still cause a diff because the order of the last two variables was different before.